resources/Materials/Examples/Syntax/PreFlattenedGraph.mtlx

```diff
@@ -0,0 +1,393 @@
+<?xml version="1.0"?>
+<materialx version="1.38" colorspace="lin_rec709" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Usually we've reserved Materials/Examples/Syntax for syntax examples in the specification, and this example might be better located in Materials/TestSuite.

Reply:
Thanks for the info. I've moved it over now to the test suite area.